charts/rollups-node/Chart.yaml

```diff
@@ -13,7 +13,7 @@ maintainers:
     url: https://github.com/oap75
 
 type: application
-version: 1.0.1-0
+version: 1.0.1-1
```

Comment:
This should be on a `DELETE:` commit, that will be handled on another specific commit bumping the version.